src/main/java/io/kestra/plugin/notifications/sendgrid/SendGridMailSend.java

```diff
@@ -157,7 +157,7 @@ public SendGridMailSend.Output run(RunContext runContext) throws Exception {
         personalization.setSubject(runContext.render(this.subject));
 
         if (this.textContent != null) {
-            Content plainTextContent = new Content(ContentType.TEXT_PLAIN.getMimeType(), runContext.render(this.htmlContent));
+            Content plainTextContent = new Content(ContentType.TEXT_PLAIN.getMimeType(), runContext.render(this.textContent));
```

Comment:
@iNikitaGricenko not sure if this PR will be merged or not.
If Not, make sense this part checks in current master we have htmlContent instead of textContent if that wasn't done especially in that way.

`new Content(ContentType.TEXT_PLAIN.getMimeType(), runContext.render(this.htmlContent));`

Reply:
As textContent can be null you should use it only if not null, if null you can either provide the HTML content as before or a default text as in the mail task.

Reply:
Please correct if I am wrong or confirm the solution:

- if textContent is null then this if will not achive even.
- If you mean that `runContext.render(this.textContent)` can be null then make sense to setting below logic
```
        if (this.textContent != null) {
            final String textContent = runContext.render(this.textContent) == null ? "Please view this email in a modern email client" : runContext.render(this.textContent);
            Content plainTextContent = new Content(ContentType.TEXT_PLAIN.getMimeType(), textContent);
            mail.addContent(plainTextContent);
        }
```

Reply:
Yes, seems good

Reply:
Handled https://github.com/kestra-io/plugin-notifications/pull/139/commits/d84fa92be782dbdafc72662731f539022174ba06